src/main/menus/file.js

```diff
@@ -1,9 +1,9 @@
-import { app } from 'electron'
```

Comment:
Aha, you maybe use JS format, But there should leave one space between `{` and `app`.